config/install/field.field.media.audio.field_media_use.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_media_use
+    - media.type.audio
+    - taxonomy.vocabulary.islandora_media_use
+  content:
+    - 'taxonomy_term:islandora_media_use:fe4c8b44-b230-45df-a7b5-62794f9b3b01'
+_core:
+  default_config_hash: _Z9k59ERH4xZEDu82qUWLlMMptINa_nX1vR8K1gKpPU
+id: media.audio.field_media_use
+field_name: field_media_use
+entity_type: media
+bundle: audio
+label: 'Media Use'
+description: 'Defined by Portland Common Data Model: Use Extension https://pcdm.org/2015/05/12/use. ''Original File'' will trigger creation of derivatives.'
```

Comment:
Extending my review, this line should be modifying [the corresponding line in the existing config](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/modules/islandora_core_feature/config/install/field.field.media.image.field_media_use.yml#L13) in the islandora module.